### PR TITLE
#48 Update H2DataTest to include update operation

### PR DESCRIPTION
**Pull request by super132**
For task #48 

- Update `H2DataTest` to add a test for `update()` method.

### CODE REVIEW

**Comment by super132** [comment]
@dmarkov Please review this PR. Thanks.

**Comment by dmarkov** [comment]
@super132 Thanks for your pull request, let me find someone who can review it

**Comment by dmarkov** [comment]
@caarlos0 review this one please

**Comment by dmarkov** [comment]
> @dmarkov Please review this PR. Thanks.

@super132 thanks, will take a look

File: src/test/java/com/jcabi/dynamo/mock/H2DataTest.java [hunk 1]
```diff
@@ -155,4 +156,36 @@ public void supportsColumnNamesWithIllegalCharacters() throws Exception {
...
+        );
+        MatcherAssert.assertThat(
+            data.iterate(
+                table, new Conditions().with(key, Conditions.equalTo(number))
+            ).iterator().next(),
+            Matchers.hasEntry(
+                Matchers.equalTo(attr),
+                Matchers.equalTo(new AttributeValue(updated))
+            )
+        );
```
Comment (caarlos0) [inline src/test/java/com/jcabi/dynamo/mock/H2DataTest.java]: @super132 maybe also assert the size of the table?

Comment (super132) [inline src/test/java/com/jcabi/dynamo/mock/H2DataTest.java]: @caarlos0 Assert condition is added, please take a look. Thanks.

**Comment by caarlos0** [comment]
@super132 thanks

**Comment by caarlos0** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by dmarkov** [comment]
@caarlos0 thank you, added **17 mins** to your acc, payment num is `AP-4DM55038UR393064F`, 93 hours and 21 mins spent in total to complete... you're getting extra minutes here (c=2)... +17 to your rating, your total score is [+2022](http://www.netbout.com/b/35095?open=rating)

**Comment by dmarkov** [comment]
@rultor please deploy

**Pull request merged**